mmv1/templates/terraform/examples/alloydb_secondary_instance_basic.tf.erb

```diff
@@ -0,0 +1,69 @@
+resource "google_alloydb_cluster" "primary" {
+  cluster_id = "<%= ctx[:vars]['alloydb_primary_cluster_name'] %>"
+  location   = "us-central1"
+  network    = google_compute_network.default.id
+  cluster_type = "PRIMARY"
+}
+
+resource "google_alloydb_instance" "primary" {
+  cluster       = google_alloydb_cluster.primary.name
+  instance_id   = "<%= ctx[:vars]['alloydb_primary_instance_name'] %>"
+  instance_type = "PRIMARY"
+
+  machine_config {
+    cpu_count = 2
+  }
+
+  depends_on = [google_service_networking_connection.vpc_connection]
+}
+
+resource "google_alloydb_cluster" "secondary" {
+  cluster_id   = "<%= ctx[:vars]['alloydb_secondary_cluster_name'] %>"
+  location     = "us-east1"
+  network      = google_compute_network.default.id
+  cluster_type = "SECONDARY"
+
+  continuous_backup_config {
+    enabled = false
+  }
+
+  secondary_config {
+    primary_cluster_name = google_alloydb_cluster.primary.name
+  }
+
+  deletion_policy = "FORCE"
+
+  depends_on = [google_alloydb_instance.primary]
+}
+
+resource "google_alloydb_instance" "<%= ctx[:primary_resource_id] %>" {
+  cluster       = google_alloydb_cluster.secondary.name
+  instance_id   = "<%= ctx[:vars]['alloydb_secondary_instance_name'] %>"
+  instance_type = google_alloydb_cluster.secondary.cluster_type
+
+  machine_config {
+    cpu_count = 2
```

Comment:
Can you make this different between the secondary and primary?

Reply:
Yes, each instance can have its own machine_config and other fields.
Making it different for each instance.

Reply:
Yes, it needs to be same during creation phase(Error is returned if not)
But can be updated after creation to a different config.
Also, added test for updating the machine config.